tests/unit_tests/test_game_resolution.py

```
@@ -7,6 +7,24 @@
     resolve_game
 )
 
+class ResolutionTest:
+    resolution: Optional[list[GameOutcome]]
```

Ruff (0.14.3)

Undefined name `Optional`

(F821)